src/main/java/project/model/dao/HTenderDAO.java

```diff
@@ -0,0 +1,140 @@
+package project.model.dao;
+
+import jakarta.persistence.TypedQuery;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Root;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import project.model.response.Initiator;
+import project.model.response.Site;
+import project.model.response.Tender;
+import project.util.HibernateSessionFactoryUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HTenderDAO implements ITenderDAO {
+    @Override
+    public Tender getByID(Integer id) {
+        try (Session session = HibernateSessionFactoryUtil.getSessionFactory().openSession()) {
+            return session.get(Tender.class, id);
+        } catch (HibernateException e) {
+            e.printStackTrace();
+        }
+        return null;
```

Comment:
returning null is a very bad idea because it may cause an application crashes during work.